### ISSUE TITLE
Accessibility for Screen Reader

### ISSUE THREAD

**Issue by ossossosso** [protz/thunderbird-conversations#1322]
Hi,
my name is Daniele from Italy.

I am a blind user with a gmail account. Even if gmail website is accessible, recently appears to be not really comfortable to navigate using keyboard and a screen reader.

For this reason I passed to Thunderbird. It is a really really good application, everything is accessible and smooth, fast and easy to use.
But honestly the conversation method to manage email was really comfortable.

I thank you so much for this add-on, but unfortunatly is not 100% accessible. Basically it means I cannot use only the keyboard to navigate list of emails inside a conversation and read the content.

Do you think this feature can be taken in consideration in the future?

Thanks a lot.

Daniele.

**Comment by Standard8** [comment]
What this could really do with is descriptions (or separate issues) of the individual accessibility issues, what isn't working and what they should be doing. That would help those that don't have the knowledge to work out what to fix.

I'd definitely like to support accessibility more, but we may need fixes from the community to help with that.

**Comment by Standard8** [comment]
Duplicate of #916

We could definitely do with some help working out what's important and what's not. Starting on the latest master would be best. Though please continue conversation in #916
